HHS/Head-Start-TTADP
# [Prod] Fix invalid "onApprovedAR" values, change "State" to "State or territory"

## Description of change

- Fix bug with session report permissions to allow file uploads when appropriate
- Add script to correct bad 'onApprovedAr' goal values.
- Change the filter "State" to "State or Territory" across the Hub.
- Adds the ability to set certain email configurations via environment variables
- Add "resource link" and "resource attachment" filters to the resource dashboard
- Prompt to preserve objectives when switching goals on the AR
- Admins can see TRs in any status
- Fix 'last tta' date

## How to test

- Check that you can upload files to a session report where you can edit it (owner, collaborator, POC)
- After running the script we should no longer have any invalid 'onApprovedAr' goal values.
- Confirm the "State" filter name has changed and still works
- Confirm the resource filters appear on the resource dashboard
- Confirm the prompt - "Keep objectives or remove objectives" appears on the AR when switching goals and that it works
- As an admin, you should be able to see a training report that's "Not Started" where you aren't attached as the owner, POC, or collaborator.
- Fix 'last tta' date so we correctly show the latest objective end date.
## Issue(s)

- https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1561
- https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1910
- https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1928
- https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1963
- https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1981

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
